### PR TITLE
 Circleci pipelines fixes

### PR DESCRIPTION
**Pull request by figuerascarlos**
- Install `gobuffalo/packr` dependency
- Change regex to match v0.0.1 and v0.0.1-rc1 version types, I split them for clarity ( Open to suggestions) 
- Remove go mod tidy

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -30,6 +30,9 @@ jobs:
     executor: go
     steps:
       - checkout
+      - run:
+          name: Install packer
+          command: go get -u github.com/gobuffalo/packr/packr
```
Comment (hrodes) [inline .circleci/config.yml:35]: we should fix an exact version not the master one

**Review by hrodes — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (fllaca) [reply .circleci/config.yml:35]: `go get` is not very handy for pulling fixed versions... you need to do some trickery with git in the $GOPATH :( . Not sure best option for us... maybe we can build a small docker image from `cirleci/golang` installing golangci-lint and packer 🤔

**Review by fllaca — APPROVED** [review]
Giving my approval here, I think this looks good enough to me and can be merged, given that Packr tool doesn't provide an easy way to install a fixed version, so I think for now it's ok using the latest version with `go get` (as a side consequence, this will make us to be up to date 🤷‍♂️ ).

Additionally, according to https://github.com/gobuffalo/packr, it seems that *packr* it will be discontinued at some point:

> NOTICE: Please consider migrating your projects to github.com/markbates/pkger. It has an idiomatic API, minimal dependencies, a stronger test suite (tested directly against the std lib counterparts), transparent tooling, and more.

The successor (https://github.com/markbates/pkger) does have a nicer release download pages, I think once we migrate to this library we can use that to use a fixed version of the tool.

**Review by hrodes — APPROVED** [review]
Let's have an small brainstorming on how to manage this kind of dependencies

**Pull request merged**